framework/src/modules/chain/transport/transport.js

```diff
@@ -321,7 +272,7 @@ class Transport {
 	async handleRPCGetSignatures() {
 		const transactions = this.transactionPoolModule.getMultisignatureTransactionList(
 			true,
-			this.constants.maxSharedTransactions,
+			this.constants.broadcasts.releaseLimit,
```

Comment:
Maybe `txReleaseLimit`?

Reply:
this was the public config value existed before.